app/models/task.rb

```diff
@@ -1,2 +1,9 @@
 class Task < ApplicationRecord
+  # after_create -> { puts " created this!!!!!!!!!#{self.id}" }
+  after_create :xyz
+
```

Comment:
add the validations for task name, date should be present. 
And display the error messages accordingly